islandora.api.php

```diff
@@ -614,7 +614,7 @@ function hook_cmodel_pid_islandora_overview_object(AbstractObject $object) {
  * Lets one alter the overview tab in object management.
  */
 function hook_islandora_overview_object_alter(AbstractObject &$object, &$output) {
-  $output = $output . drupal_render(drupal_get_form('some_form', $object));
+  $output = $output . drupal_render(drupal_get_form('some_form', $object)));
```

Comment:
Also, this is introducing a syntax error (assuming this was "real" code), no? (two opening parentheses, three closing).